SPARCoreTests/SPARCoreTests.m

```diff
@@ -43,7 +41,7 @@ - (void)testJsonParser {
     XCTAssertEqualObjects(expectedArticle.articleId, returnValues[0].articleId);
     XCTAssertEqualObjects(expectedArticle.title, returnValues[0].title);
 }
-
+*/
```

Comment:
See above comment about why this test is failing... Please don't comment out failing tests; they're failing for a reason!

(exception being the commented block below because it was incorrect, not because it failed)